### ISSUE TITLE
consider removing a couple of css rules

### ISSUE THREAD

**Issue by anointed** [themeblvd/jumpstart#15]
In my current child theme, I am doing like I do with many themes, I am using the menu css attribute to assign background images to the main menu item. (screen options in menu admin, then click css button to turn on custom menu items css)

an example line from my custom.css file where I add a background image to a menu item:

```
.menu-airlines > a { background-image: url("images/primary-menu/menu-airlines.png"); }
```

Problem:
The default css file contains a couple of attributes which are virtually impossible to override in a child theme:

```
#primary-menu > li.current_page_item > a { background: none repeat scroll 0 0 #FFFFFF; }

#primary-menu > li > a:hover { background: none repeat scroll 0 0 #F2F2F2; }
```

What happens is of course if I am on the current menu item, or if I hover over a menu item, them my custom menu background images are of course not being displayed.

If I had the same background image on every single menu item, then of course it would be easy to override via custom.css, but because each menu item has its own custom background image, this is not possible.

Basically, having these 2 rules in the default css, makes using background css images on the primary menu impossible to do properly without hacking the core primary.css file.
